src/makefun/main.py

```diff
@@ -197,6 +197,9 @@ def create_function(func_signature,             # type: Union[str, Signature]
     if module_name is None:
         module_name = getattr_partial_aware(func_impl, '__module__', None)
 
+    # lambda check
+    is_lambda = func_name == (lambda: None).__name__
```

Comment:
Would we instead rely on an independent util `is_lambda_function(f)` ? It would be easier to maintain in the long run.

Also `__code__.co_name` seems more reliable than `f.__name__` as it cannot be modified. See one of the comments in  https://stackoverflow.com/a/23852434/7262247